dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -451,5 +451,9 @@ error.extendsAlias.collision=CWWKG0100E: The {2} ibm:extendsAlias attribute is d
 error.extendsAlias.collision.explanation=The ibm:extendsAlias attribute must be unique within a single extends hierarchy.
 error.extendsAlias.collision.useraction=Make each ibm:extendsAlias attribute unique within the extends hierarchy.
 
+warning.binding.config=CWWKG0101W: The configuration with the persisted identity (PID) {0} will be bound to the bundle {1}. The configuration will not be visible to other bundles.  
+warning.binding.config.explanation=A configuration is bound to the calling bundle when the getConfiguration method of the ConfigurationAdmin interface is called for the first time. Once the configuration is bound, it is no longer visible to other bundles.  
+warning.binding.config.useraction=Use the getConfiguration method on the ConfigurationAdmin interface that takes a bundle location as the second parameter. This ensures that the bundle location will always be the same expected value. 
```

Comment:
Suggested change:
To ensure that the bundle location is the same expected value, use the getConfiguration method on the ConfigurationAdmin interface that takes a bundle location as the second parameter.